evm/vm/computation.py

```diff
@@ -188,6 +194,12 @@ def add_log_entry(self, account, topics, data):
         validate_is_bytes(data, title="Log entry data")
         self.log_entries.append((account, topics, data))
 
+    def get_opcode_fn(self, opcode):
+        try:
+            return self.vm.opcodes[opcode]
```

Comment:
I'm curious as to why you think it's better encapsulated in the Computation class since it has to look up the opcode by accessing a vm attribute.

Reply:
Now that you point it out, I'm not sure why I did this.  Going to close this issue.